src/employees/entities/employees.entity.ts

```diff
@@ -1,17 +1,57 @@
-import { Entity, Column, PrimaryGeneratedColumn, Generated } from 'typeorm';
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  Generated,
+  CreateDateColumn,
+  UpdateDateColumn,
+  OneToMany,
+} from 'typeorm';
+import { Employee_Saving } from './employee_saving.entity';
 
 @Entity()
 export class Employees {
```

Comment:
its not plural. so class `Employees `, but create tables name as plural `employees`, @Entity("employees")

Reply:
done